api/lib/objectUtilities.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Checks if nested properties exists on an object.
+ *
+ * Ref: https://stackoverflow.com/questions/2631001/test-for-existence-of-nested-javascript-object-key
+ *
+ * @see unit tests
+ * @param object
+ * @returns {boolean}
+ */
+export function propsExist(object, nestedProps) {
```

Comment:
There's an "exists" function within the other projects that does this too, except also handles dot notation, e.g. `exists(services, 'idam.client')`

Reply:
that functionality can be easily added to this method with the following, should we want to:

```
if (typeof nestedProps === 'string') {
  nestedProps = nestedProps.split('.');
}
```